Lib/site.py

```diff
@@ -75,6 +75,8 @@
 import _sitebuiltins
 import io
 
+is_pypy = '__pypy__' in sys.builtin_module_names
```

Comment:
Is there any reason for a public attribute? How about an internal attribute instead?

```suggestion
_is_pypy = '__pypy__' in sys.builtin_module_names
```

Reply:
Adopted `_is_pypy`